content/blog/parsnip-0-1-2/index.Rmd

```diff
@@ -0,0 +1,279 @@
+---
+output: hugodown::hugo_document
+
+slug: parsnip-0-1-2
+title: parsnip 0.1.2
+date: 2020-07-09
+author: Max Kuhn
+description: >
+    A new version of parsnip bring improvements in how predictors are handled and
+    a few other neat features.  
+
+photo:
+  url: https://unsplash.com/photos/kJxinkriuB4
+  author: Kevin Lanceplaine
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [package] 
+tags: [parsnip, tidymodels,]
+---
+
+We're happy to announce the release of [parsnip](https://parsnip.tidymodels.org/) 0.1.2. parsnip is a unified tidy interface to many modeling techniques. 
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("parsnip")
+```
+
+You can see a full list of changes in the [release notes](https://parsnip.tidymodels.org/news/index.html). I'll highlight the big changes here. The primary improvement is related to the brand new versions of the [hardhat](https://hardhat.tidymodels.org/news/index.html) and [workflows](https://workflows.tidymodels.org/news/index.html) packages. 
+ 
+```{r setup, include = FALSE}
+library(tidymodels)
+library(modeldata)
+library(ranger)
+library(rpart)
+library(ggparty)
+library(partykit)
+data(penguins)
+
+thm <- theme_bw() + 
+  theme(
+    panel.background = element_rect(fill = "transparent", colour = NA), 
+    plot.background = element_rect(fill = "transparent", colour = NA),
+    legend.position = "top",
+    legend.background = element_rect(fill = "transparent", colour = NA),
+    legend.key = element_rect(fill = "transparent", colour = NA)
+  )
+theme_set(thm)
+```
+
+
+## Predictor encoding consistency
+
+Normally, when using giving a formula to an R modeling function, the standard `model.matrix()` machinery will convert factor predictors to a set of binary indicator columns However, there are a few notable exceptions: 
+
+* Tree-based models, such as CART, C5.0, or random forests, don't require binary indicator variables since their splitting methods can create groups of categories. For a function like `ranger::ranger()`, the formula does _not_ create indicators; the factor predictors are left as factors. 
```

Comment:
```suggestion
* Tree-based models, such as CART, C5.0, and random forests, don't require binary indicator variables since their splitting methods can create groups of categories. When using a tree-based model function like `ranger::ranger()`, the formula does _not_ create indicators; the factor predictors are left as factors. 
```